src/main/java/br/unicamp/cecom/appointmentscheduler/core/exception/NotFoundException.java

```diff
@@ -0,0 +1,19 @@
+package br.unicamp.cecom.appointmentscheduler.core.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class NotFoundException extends RuntimeException {
+
+    private static final long serialVersionUID = -4438072051802392231L;
```

Comment:
doubt: This is a leftover right? If so, please remove it :)

Reply:
https://pt.stackoverflow.com/questions/96903/o-que-significa-serialversionuid

Reply:
reference: https://pt.stackoverflow.com/questions/96903/o-que-significa-serialversionuid